Framework/Muon/test/MuonGroupingAsymmetryTest.h

```diff
@@ -336,6 +336,33 @@ class MuonGroupingAsymmetryTest : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(wsOut->readE(0)[0], 0.0002351, 0.00001);
     TS_ASSERT_DELTA(wsOut->readE(0)[1], 0.0002460, 0.00001);
   }
+
+  void
+  test_grouping_asymmetry_with_specified_normalization_gives_correct_values() {
+    auto ws = createMultiPeriodAsymmetryData(3, 2, 10, "group_asym");
+    std::vector<int> detectors = {1, 2};
+    auto alg =
+        setUpAlgorithmWithoutOptionalProperties(ws, "group_asym", detectors);
+
+    std::vector<int> summedPeriods = {3, 2};
+    std::vector<int> subtractedPeriods = {1};
+    alg->setProperty("SummedPeriods", summedPeriods);
+    alg->setProperty("SubtractedPeriods", subtractedPeriods);
+    alg->setProperty("NormalizationIn", 15.0);
+    alg->execute();
+
+    auto wsOut = getOutputWorkspace(alg);
+
```

Comment:
should also assert that the norm is 15, we can get the value from the sample logs of the output